libsrc/atari2600/crt0.s

```diff
@@ -0,0 +1,46 @@
+; Atari VCS 2600 startup code for cc65
+;
+; Florent Flament (contact@florentflament.com), 2017
+
+        .export         _exit
+        .export         __STARTUP__ : absolute = 1
+
+        .import         __RAM_START__, __RAM_SIZE__
+        .import         copydata
+        .import         _main
+
+        .include        "zeropage.inc"
+
+
+.segment "STARTUP"
+start:
+; Clear decimal mode
+        cld
+; Clear zeropage & registers
+; Also Initializes system stack pointer
+        ldx     #0
+        txa
+clearLoop:
+        dex
+        txs
+        pha
+        bne     clearLoop
+
+; Initialize data
```

Comment:
The initialization loop clears the whole (128 bytes of) memory with 0s (including the BSS segment). I don't see the advantage of calling `zerobss` (unless it does something else than writing zeros to the BSS segment, that I've not seen).